File: test_kontests.py
import requests

def test_kontests():
    url = "https://kontests.net/api/v1/codeforces"
    print(f"Testing {url}...")
    try:
        res = requests.get(url, timeout=10)
        print(f"Status: {res.status_code}")
        if res.status_code == 200:
            data = res.json()
            print(f"Found {len(data)} contests")
            if len(data) > 0:
                print("First contest:", data[0])
    except Exception as e:
        print(f"Error: {e}")

if __name__ == "__main__":
    test_kontests()

File: test_kontests.js
const https = require('https');

const url = 'https://kontests.net/api/v1/codeforces';

console.log(`Fetching ${url}...`);

https.get(url, (res) => {
    console.log('statusCode:', res.statusCode);

    let data = '';
    res.on('data', (d) => {
        data += d;
    });

    res.on('end', () => {
        console.log('Body length:', data.length);
        if (res.statusCode === 200) {
            try {
                const json = JSON.parse(data);
                console.log('First contest:', json[0]);
            } catch (e) {
                console.log('Error parsing JSON:', e);
                console.log('Body:', data.substring(0, 200));
            }
        } else {
            console.log('Body:', data.substring(0, 200));
        }
    });

}).on('error', (e) => {
    console.error(e);
});
